### ISSUE TITLE
Change update date's

### ISSUE THREAD

**Issue by ModernMoGamer** [SmartlyDressedGames/Unturned-3.x-Community#869]
TL:DR at the end

Updates are great, but are upsetting when they break the game or servers to a point where they require development and trouble shooting involvement.

I've found a few updates in the recent months to require human intervention to fix some of the update issues, when this happens during the troubleshooting phase, the server can be broken to a point in which the players are unable to have a quality experience.

Thus when players don't have their quality experience, they head over to either another server or game. Then for the remainder of the weekend the server has lower than normal player population.

Changing the usual updates that happen on Friday to a Monday or Tuesday would be much more beneficial in my opinion to prevent bugs from killing the weekend on servers.

TL:DR:
Change when the update's come out on Fridays and change it to Monday's or Tuesday's to prevent a weekend of lower player count after bugs hold the server from players.

Thank you very much for reading my post!

**Comment by Mithallas** [comment]
I would be surprised if nelson didn't already consider this, possibly some other factors.

To play devil's advocate, releasing large updates such as new maps or content on Friday generates hype for the players which is a good way to start the weekend

Although June 21st's update was great for server admins (thnx Nelson! 👍 ); not quite as hype for players. Maybe would have been better on a Monday so debug wouldn't have been over the weekend.

**Comment by SDGNelson** [comment]
This past update has been especially rocky, 4 patches is pretty bad. Base-game-wise I'm happy with how stable it was, but there've been a lot of cases with custom content to troubleshoot the past few days.

I think it will be a while before another big update like this. In particular I'm hoping to do the 2018.4 LTS update, more workshop improvements, and misc other requests from here all in one update which will go on preview for a while. For that it'd especially make sense to do an update earlier in the week.

**Comment by Johnanater** [comment]
Friday updates are good for hype and stuff, but a bad update could cause trouble for weekend traffic. As Nelson said it's very rare, so I think it should stay Friday
